Webex Connect Flows/v3.0/Template/Event Handling Workflows/README.md

```diff
@@ -0,0 +1,155 @@
+output variables section.
+
+If you want to run the Routed Event flow to run only for specific asset or channel then we need to enable the conditions checkbox and add condition to the "WxCC Task V2" Node.
+For adding conditions please refer to the [Adding Conditions In Flow](#adding-conditions-in-flow) section.
+
+There is no restriction on the number of Routed Event Flow. You can add as many Routed Event Flow you need if different handling is required for each channel or asset.
+But in this case you need to specify condition for each flow.
+
+As webex.variables and webex.RequestBody are nested variables , we are assigning these to response and requestBody respectively so that we can use them further.
+
+<img width="800" alt="WxCC Task v2 Transition Actions" src="../../images/WxCC Task v2 Transition Actions.png">
+
+### Modified Event flow
+
+* This workflow will be triggered when an agent has done a conversation transfer request or a conference request. The event for this workflow is 'Task Modified'. So once the contact has been transferred to another agent or
+  the contact has been conferenced with another agent then this workflow will be triggered.
+
+#### When do you need Modified Event Flow ?
+* Modified event flow is an optional flow. It is needed only when you want to do some customization like Screen Pop or http calls after the contact has been transferred or conferenced.
+
+#### Start Node of Modified Event Flow
+
+<img width="800" alt="ModifiedTaskv2" src="../../images/TaskModified.png">
+
+Under the configuration section , we can see the event selected is Task Modified so this flow will be triggered on the Task Modified event. We can use the available variables from the
+output variables section.
+
+If you want to run the Modified Event flow to run only for specific asset then we need to enable the conditions checkbox and add condition to the "WxCC Task V2" Node.
+For enabling conditions please refer to the [Adding Conditions In Flow](#adding-conditions-in-flow) section.
+
+
+There is no restriction on the number of Modified Event Flow. You can add as many Modified Event Flow you need if different handling is required for each channel or asset.
+But in this case you need to specify condition for each flow.
+
+As webex.variables and webex.RequestBody are nested variables , we are assigning these to response and requestBody respectively so that we can use them further.
+
+<img width="800" alt="WxCC Task v2 Transition Actions" src="../../images/WxCC Task v2 Transition Actions.png">
+
+
+##### Branch node
+
+<img width="800" alt="Branch" src="../../images/Branch.png">
+
+In the branch we are specifying if "webex.context" is add or remove that means if we are trying to add or remove participant from the
+contact then we can proceed further with the flow. In this sample we are showing screen pop if the "webex.context" is add or remove.
```

Comment:
Try to use less pronouns wherever possible to sound more formal - 

In the branch node, condition is being checked for "webex.context". Context variable specifies if an agent/participant is added or removed from the contact. In this sample screen pop is shown if the "webex.context" is one of "add" or "remove".